server/testsupport/prisonerTestDataBuilder.ts

```diff
@@ -1,14 +1,23 @@
 import type { Prisoner } from 'prisonRegisterApiClient'
 
-export default function aValidPrisoner(prisonNumber = 'A1234BC', prisonId = 'BXI'): Prisoner {
+export default function aValidPrisoner(options?: {
```

Comment:
I changed this test builder function to use the approach of passing in an `options` map seeing as I was in the area 👍 
We'll do the others as and when we get to them 👍